Gemfile

```diff
@@ -3,3 +3,7 @@ source "https://rubygems.org"
 gem "cocoapods", "~> 1.11.3"
 gem "fastlane", "~> 2"
 gem "xcpretty", "~> 0"
+
+
+plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
+eval_gemfile(plugins_path) if File.exist?(plugins_path)
```

Comment:
@staszekscp I think we should ensure there is new line at the end of all files, that is our guide.

Reply:
+1

Reply:
Sure! I haven't noticed that!